en/docs/get-started/tutorials/create-your-first-webhook.md

```diff
@@ -31,151 +31,42 @@ First, let's add a Webhook component as follows:
     | **Name**        | `IssueTracking`       |
     | **Description** | `Track GitHub issues` |
 
-4. Click **Create**.
+4. Click **Create**.This takes you to the **Components** page.
 
-    The **Components** page opens as shown in the below image.
+5. On the **Components** page, click **Create** on the the **Webhook** card.
 
-    ![Create component](../../assets/img/tutorials/rest-api/create-component.png){.cInlineImage-full}
+    ![Create component](../../assets/img/tutorials/webhook/create-webhook.png){.cInlineImage-small}
 
-5. On the Components page, click **+ Create**.
+6. Provide authorization with your GitHub account by clicking  **Authorize with GitHub**. 
+7. If you have not already authorized Choreo apps, click **Authorize Choreo Apps** when prompted.
+8. Select a GitHub account, a repository that includes a Ballerina project or a dockerfile, the relevant branch and the build preset as **Ballerina**. 
+9. Enter a valid path relative to the root of your repository that points to the implementation of the webhook. If you have not designed and implemented your webhook yet, you can connect an empty repository or a sub-folder and proceed to create the component. 
```

Comment:
Change "empty repository" to "empty GitHub repository".

Reply:
done

Reply:
This is not yet done.